app/serializers/articles_index_serializer.rb

```diff
@@ -1,5 +1,5 @@
 class ArticlesIndexSerializer < ActiveModel::Serializer
-  attributes :id, :title, :teaser, :date, :category
+  attributes :id, :title, :teaser, :date, :category, :image
```

Comment:
Are we testing this change? What's the value of image?

Reply:
I am not sure about it to be honest. I think image will be an object. In test environment if we try to get to url in serializer (like in PT), all tests fail as it cannot get attribute url of image, but in Rails console url is accessible in article.image.url. I am not really sure what to think about it, probably need to spin up server, create an article, than get one, and look at response. Can do it after dinner